pandas/core/frame.py

```diff
@@ -254,6 +255,29 @@
 3  foo        5  foo        8
 4  bar        2  bar        6
 5  baz        3  baz        7
+
+Merge DataFrames A and B. Specify the left and right suffix
+to append to the name of any overlapping columns.
+
+>>> A.merge(B, left_on='lkey', right_on='rkey', how='outer',
```

Comment:
sorry, I didn't see before that in the original docstring the DataFrame names were `A` and `B`. Do you mind changing them to `df1` and `df2`. That's the standard we use, and `A` and `B` in Python should be used for classes, not instances.

Also, we can take care of that in a different PR if you prefer, but it'd be great to have a more real world example (it makes it easier to understand what's going on), and in this case I find the `how='outer'` misleading, as it doesn't add value.

Reply:
Yes, I can make those updates.

Reply:
Also thanks for the location of the validation scripts and such.  This is very helpful for my first documentation pull request.